spec/changelog.yml

```diff
@@ -7,6 +7,9 @@
   - description: Prepare for next patch release
     type: enhancement
     link: https://github.com/elastic/package-spec/pull/432
+  - description: Allowing multiple services included as part of custom-agent-deployer
```

Comment:
As a follow up of this change, please open a PR to update the `elastic-agent` documentation: https://github.com/elastic/elastic-package/blob/main/docs/howto/system_testing.md#agent-service-deployer